kong/api/init.lua

```diff
@@ -28,7 +28,7 @@ ngx.log(ngx.DEBUG, "Loading Admin API endpoints")
 
 
 -- Load core routes
-for _, v in ipairs({"kong", "health", "cache", "config", "clustering"}) do
+for _, v in ipairs({"kong", "health", "cache", "config", }) do
```

Comment:
Why was this not renamed and removed?
Will the code actually pick up the endpoints defined in `cluster_status` if it is not included in this list?

Reply:
This is not necessary for DAO routes, they are automatically loaded by Kong. For example, `kong/api/routes/plugins.lua` is also not in the above list.